### PR TITLE
Add postgres-veritable as dependency of veritable-cloudagent

### PR DESCRIPTION
**Pull request by jonmattgray**
The new postgres for `veritable-cloudagent` was missing from docker compose so `npm run compose:up:deps:test` wasn't bringing it up. See tests failing with cloudagent unavailable https://github.com/digicatapult/nice-agent-portal/actions/runs/9061988059/job/24894885760?pr=71

Also removes some envs that I think aren't used?

### CODE REVIEW

File: .env.nice-agent-alice.dev [hunk 1]
```diff
@@ -3,5 +3,3 @@ DB_PORT=3020
 CLOUDAGENT_PORT=3010
 PORTAL_BASE_URL=http://host.docker.internal:3000
 DATABASE_URL=postgresql://postgres:postgres@localhost:3020/nice-agent-portal
-VERITABLE_PG_HOST="localhost"
```
Comment (msterle) [inline .env.nice-agent-alice.dev:6]: I think we need these env vars for running in "development mode", where the dependencies are run through compose, but one of the agents is run on the host machine

**Review by msterle — CHANGES_REQUESTED** [review]

File: docker-compose.yml
Comment (Ellenn-A) [inline docker-compose.yml:1]: this looks good, Sorry I missed that. But agree wt Mike I don't know why we are removing the envs for `.env...dev` files

File: .env.nice-agent-alice.dev (hunk 1 above)
Comment (jonmattgray) [reply .env.nice-agent-alice.dev:6]: in dev mode `veritable-cloudagent` + its postgres still runs in docker so I don't see why the host would be different? It's only `nice-agent-portal` running on the host machine, and that doesn't talk to `veritable-cloudagent`s postgres

**Review by msterle — APPROVED** [review]
lgtm!

**Pull request merged**